test/fixtures/test-runner/output/arbitrary-output.js

```diff
@@ -0,0 +1,20 @@
+// Flags: --test --expose-internals
+'use strict';
+
+const v8_reporter = require('internal/test_runner/reporter/v8-serializer');
+const { Buffer } = require('buffer');
+
+
+(async function () {
+  const reported = v8_reporter([
+    { type: "test:pass", data: { name: "test", nesting: 0, file: __filename, testNumber: 1, details: { duration_ms: 0 } } }
+  ]);
+
+  for await (const chunk of reported) {
+    process.stdout.write(chunk);
+    process.stdout.write(Buffer.concat([Buffer.from("arbitrary - pre"), chunk]));
+    process.stdout.write(Buffer.from("arbitrary - mid"));
+    process.stdout.write(Buffer.concat([chunk, Buffer.from("arbitrary - post")]));
+  }
+})();
```

Comment:
Should we have a `.then(common.mustCall())` here?

Reply:
no need, we snapshot the output